JS/Lesson34/goksel/recipe-app/src/modules/recipes/NewRecipe/NewRecipeForm/index.jsx

```diff
@@ -0,0 +1,233 @@
+                )}
+            </Grid>
+
+            <Grid item>
+                <TextField
+                    sx={{ minWidth: {xs:300, sm:400, md: 500, lg:600}}}
+                    id="strMealThumb"
+                    label="Share Your Image Link"
+                    variant="outlined"
+                    size="small"
+                    {...register("strMealThumb")}
+                />
+            </Grid>
+
+            <Grid item>
+                <FormControl sx={{ minWidth: {xs:300, sm:400, md: 500, lg:600}}}>
+                    <InputLabel id="strArea-label">Meal Area</InputLabel>
+                    <Select
+                        size="small"
+                        labelId="strArea-label"
+                        id="strArea"
+                        label="Meal Area"
+                        {...register("strArea", { required: true })}
+                    >
+                        {MealArea.map((area) => (
+                            <MenuItem value={area.title} key={area.id}>{area.title}</MenuItem>
+                        ))}
+                    </Select>
+                </FormControl>
+            </Grid>
+
+            <Grid item>
+                <TextField
+                    sx={{ minWidth: {xs:300, sm:400, md: 500, lg:600}}}
+                    id="strSource"
+                    label="Source of Your Meal"
+                    variant="outlined"
+                    size="small"
+                    {...register("strSource")}
+                />
+            </Grid>
+
+            <Grid item>
+                <TextField
+                    sx={{ minWidth: {xs:300, sm:400, md: 500, lg:600}}}
+                    id="strDrinkAlternate"
+                    label="Drink Alternate"
+                    variant="outlined"
+                    size="small"
+                    {...register("strDrinkAlternate")}
+                />
+            </Grid>
+
+            <Grid item>
+                <TextField
+                    sx={{ minWidth: {xs:300, sm:400, md: 500, lg:600}}}
+                    id="strImageSource"
+                    label="Source of Your Image"
+                    variant="outlined"
+                    size="small"
+                    {...register("strImageSource")}
+                />
+            </Grid>
+
+            <Grid item>
+                <TextField
+                    sx={{ minWidth: {xs:300, sm:400, md: 500, lg:600}}}
+                    id="strYoutube"
+                    label="Share Youtube Link"
+                    variant="outlined"
+                    size="small"
+                    {...register("strYoutube")}
+                />
+            </Grid>
+
+            <Grid item>
+                <FormControl sx={{ minWidth: {xs:300, sm:400, md: 500, lg:600}}}>
+                    <InputLabel id="strCreativeCommonsConfirmed-label">Is Creative Commons Confirmed</InputLabel>
+                    <Select
```

Comment:
I think it would be better to use Radio input here

Reply:
I added radio.